bypros/test/docker-entrypoint-initdb.d/schema.sql

```diff
@@ -0,0 +1,394 @@
+--
```

Comment:
Should there be a
```
echo schema.sql binary >> .gitattributes
```
here?

Reply:
no, the schema is plain text

Reply:
It's just to remove it from the git-diff. Makes it more readable, specifically if it's machine-generated.